countdown/builder.js

```diff
@@ -0,0 +1,164 @@
+"use strict";
+const e = React.createElement;
+const baseUrl = "https://aminorgames.com/stream_tools/countdown/";
```

Comment:
Is there a way we can do this without hard-coding the complete URL? I have a "preview" place I work where I can test things out without potentially disrupting what is now a service a couple others are relying on.